quinn-udp/src/unix.rs

```diff
@@ -143,7 +143,7 @@ fn init(io: SockRef<'_>) -> io::Result<()> {
     Ok(())
 }
 
-#[cfg(not(any(target_os = "macos", target_os = "ios")))]
+#[cfg(not(any(target_os = "macos", target_os = "ios", target_os = "android")))]
```

Comment:
This seems like a pretty heavy hammer, which impairs performance for all Android builds on account of what's presumably a pretty small minority of very old phones.

Reply:
I'd prefer the dynamic fallback. It's a common enough Linux pattern, and lets us avoid baking in assumptions about use cases. I wouldn't be surprised if we ended up needing similar tricks on other platforms sooner or later either.

Even for typical mobile applications where CPU might not be a bottleneck, power use is still important, for example.